src/types.js

```diff
@@ -20,6 +20,7 @@ export type Options = {
   noColor?: boolean,
   testNamePattern?: string,
   testFileNamePattern?: string,
+  reporters?: string,
```

Comment:
I think you might want to take an array, see [jest cli](https://jestjs.io/docs/en/cli#--reporters).

also please update `index.d.ts`.

Reply:
Done. Thanks for pointing that out.